purchase_product_weight/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Sergio Corato <https://github.com/sergiocorato>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+{
+    'name': 'Purchase product on weight',
+    'version': '10.0.1.0.0',
+    'category': 'Purchase',
+    'license': 'LGPL-3',
+    'author': 'Sergio Corato, Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/purchase-workflow',
+    'depends': [
+        'purchase',
+    ],
+    'data': [
+        'views/purchase.xml',
+        'views/product.xml',
+        'views/product_supplierinfo.xml',
+    ],
+    'installable': True,
```

Comment:
Do you plan to be maintainer of this module, you can,
'maintainers': ['your_git_login'],

Reply:
:+1: